src/cobertura.rs

```diff
@@ -491,7 +491,14 @@ pub fn output_cobertura(
         .unwrap();
 
     let result = writer.into_inner().into_inner();
-    let mut file = BufWriter::new(get_target_output_writable(output_file));
+    let output_file = output_path.map(|path| {
+        if path.is_dir() {
+            path.join("cobertura.xml")
```

Comment:
What if we do this in main, using the output format name? This way we don't have to repeat this code in each output format function

Reply:
pushed